src/services/dev.js

```diff
@@ -9,4 +9,8 @@ export default {
       language: 'en',
     });
   },
+  async getChat() {
+    await utils.delay(50);
+    return utils.camelizeKeys([]);
```

Comment:
it's a mock api, so better return some meaningful data rather than an empty array?